pkg/controller/metadata/metadata_controller.go

```diff
@@ -58,6 +58,7 @@ const (
 	validUntil         = "validUntil"
 	beginTag           = "-----BEGIN CERTIFICATE-----\n"
 	endTag             = "\n-----END CERTIFICATE-----"
+	requeueAfterNS     = 1800000000000
```

Comment:
Is there a nicer more human readable way this can be represented?

Reply:
Just divide by 60000000000 to get number of minutes ;)

Reply:
There might be. I'm going to leave as is for now. The whole thing needs refactoring so will save it for then...